common/utilities/integration_adapters_logger.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+import time
+
+AUDIT = 25
+
+
+def _add_audit_level_log():
+    """
+    Adds a new log level to the standard python logging module, this level is the `audit` level and is defined as
+    a high level than INFO but lower than  WARNING
+    """
+    logging.addLevelName(AUDIT, "AUDIT")
+
+    def audit(self, message, *args, **kws):
+        if self.isEnabledFor(AUDIT):
+            self._log(AUDIT, message, args, **kws)
+
+    logging.Logger.audit = audit
+
+
+def load_global_log_config():
+    """
+    A general method to load the overall config of the system, specifically it modifies the root handler to output
+    to sysout and sets the default log levels and format. This is expected to be called once at the start of a
+    system.
+    """
+    _add_audit_level_log()
+    logger = logging.getLogger()
+    logger.setLevel(logging.NOTSET)
+    handler = logging.StreamHandler(sys.stdout)
+    logging.Formatter.converter = time.gmtime
+    formatter = logging.Formatter('[%(asctime)s.%(msecs)03d] '
+                                  '%(message)s pid=%(process)d LogLevel=%(levelname)s ProcessKey=%(processKey)s',
+                                  '%Y-%m-%d %H:%M:%S')
```

Comment:
I think it's worth using the `T` separator between date and time, as specified in the [logging principles document](https://gpitbjss.atlassian.net/wiki/spaces/RTDel/pages/1560608910/KDD-001+-+Logging+Strategy+and+Format#KDD-001-LoggingStrategyandFormat-Timestamp), [since it appears that omitting it needs to be something both parties agree on](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations):

```suggestion
                                  '%Y-%m-%dT%H:%M:%S')
```